lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -45,11 +46,15 @@ def ask_question_loop
     end
 
     def activate
+      if key_exist? && force
+        backup_key
+      end
+
```

Comment:
Oh. Sorry I misunderstand your means. If the backup is created by this program, it will never exist after a success or a fail fetch. If it's backup by user manually with the `v2_key.backup` name, hmm, this `backup_key` call will overwrite user's backup, that's a problem